### DISCUSSION TITLE
How to stop slide immediately on hover?

### DISCUSSION THREAD

**Discussion by glassesUSA** [nolimits4web/swiper#4672] [Q&A]
How can I stop the swiper from moving immediately after hover. Currently it takes speed into account. I've tried doing

a. Setting transition to '0ms'
b. Changing the speed as you hover

But nothing seems to work... any ideas?

**Comment by glassesUSA** [comment]
@nolimits4web 

Is there any way in the future that you can implement this?

**Reply by nolimits4web** [reply]
You also need to get current translate and set it:
```
const currentTranslate = swiper.getTranslate()
swiper.setTransition(0)
swiper.setTranslate(currentTranslate)
```

**Reply by warudin** [reply]
The above works, but I can't get the slider to move again after leaving the slider.
I am trying to implement this on a slider with continuous movement.
Any tips on that?

Here's an example of what I've got so far:
https://codesandbox.io/s/swiper-slides-per-view-forked-pb1oun?file=/index.html
